pkg/nsx/services/subnet/builder.go

```diff
@@ -44,6 +44,13 @@ func (service *SubnetService) buildSubnetSetName(subnetset *v1alpha1.SubnetSet,
 	return util.GenerateTruncName(common.MaxSubnetNameLength, resName, "", index, "", "")
 }
 
+func convertAccessMode(accessMode string) string {
```

Comment:
we do not need this func. NSX VPC subnet has the below access mode, in Subnet/SubnetSet CR, we should keep the same access mode.
access_mode:
      type: string
      enum:
        - Private
        - Public
        - Isolated
        - Private_TGW